guide/src/features.md

```diff
@@ -28,6 +28,27 @@ These features are an extension of the `abi3` feature to specify the exact minim
 
 See the [building and distribution](building_and_distribution.md#minimum-python-version-for-abi3) section for further detail.
 
+### `serde`
```

Comment:
This needs to go in the "Advanced Features" section (which should maybe be renamed to "Other Features") as this feature could be useful in the embedding case too.

Reply:
fixed, thanks!